v2/pkg/protocols/dns/dns.go

```diff
@@ -198,3 +200,19 @@ func classToInt(class string) uint16 {
 	}
 	return uint16(result)
 }
+
+func generateDNSVariables(domain string) map[string]interface{} {
+	parsed, err := publicsuffix.Parse(strings.TrimSuffix(domain, "."))
+	if err != nil {
+		return map[string]interface{}{"FQDN": domain}
+	}
+
+	domainName := strings.Join([]string{parsed.SLD, parsed.TLD}, ".")
```

Comment:
instead of creating a slice and iterate it to concatenate 3 strings, wouldn't have been easier to use `fmt.Sprintf` instead?

Reply:
Just another way of doing the same thing :D